pyfarm/scheduler/tasks.py

```diff
@@ -228,10 +228,27 @@ def send_tasks_to_agent(self, agent_id):
                     task.attempts -= 1
                     db.session.add(task)
                 db.session.commit()
+            if response.status_code == requests.codes.conflict:
```

Comment:
Should we log an error or warning if `rejected_task_ids` is not in the response data (and log what agent it came from)?  The first logger.error call below in combination with `CONFLICT` response would seem to indicate there was in fact a conflict.  Not actually finding rejected ids to iterate over would either be a bad behavior on the agent, an out dated agent or something else we didn't expect.